srcpkgs/libblockdev/template

```diff
@@ -1,22 +1,22 @@
 # Template file for 'libblockdev'
 pkgname=libblockdev
-version=2.28
+version=3.0.3
 revision=1
 build_style=gnu-configure
 hostmakedepends="pkg-config python3"
 makedepends="cryptsetup-devel device-mapper-devel dmraid-devel libbytesize-devel
  libglib-devel libkmod-devel libparted-devel nss-devel
- volume_key-devel libyaml-devel pcre2-devel"
+ volume_key-devel libyaml-devel pcre2-devel libfdisk-devel
+ e2fsprogs-devel libnvme-devel keyutils-devel"
 short_desc="Library for manipulating block devices"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="https://github.com/storaged-project/libblockdev"
 # changelog needs to be adjusted on major version changes
-changelog="https://raw.githubusercontent.com/storaged-project/libblockdev/2.x-branch/NEWS.rst"
+changelog="https://raw.githubusercontent.com/storaged-project/libblockdev/master/NEWS.rst"
 distfiles="https://github.com/storaged-project/libblockdev/releases/download/${version}-1/libblockdev-${version}.tar.gz"
-checksum=82c9c841e28a74fecadedebfae6a772df623cecdf652e5376650fa26da5b7df4
-conf_files="/etc/libblockdev/conf.d/10-lvm-dbus.cfg
- /etc/libblockdev/conf.d/00-default.cfg"
+checksum=0f2872830293f3e222832903d80de4c38e06a9c3ea18915b3263272a72095ea8
+conf_files="/etc/libblockdev/3/conf.d/10-lvm-dbus.cfg /etc/libblockdev/3/conf.d/00-default.cfg"
```

Comment:
this change will result in the previous conf_files being removed even if modified. it may be better to change blockdev to look for these files not in the `3` dir

Reply:
I originally left it the same as what is already there (i.e, looks in `/etc/libblockdev/conf.d`) but it ran into a "configuration file could not be found in DESTDIR" error.